test/mocha/core/oauth2-authorize.js

```diff
@@ -20,6 +20,18 @@ describe("oauth2", function () {
   }
 
   describe("authorize redirect", function () {
+    let initialGenerateCodeVerifier,
+        initialCreateCodeChallenge
+
+    beforeEach(() => {
```

Comment:
Not doing resulted in the methods being mocked for all the other tests. Do you have any idea how I could do it differently?

Reply:
Yes! `expect` has a [`spyOn(target, "method")` function](https://github.com/mjackson/expect#spyon).

I've switched this PR over to it in 898bfc26b.

Reply:
That is nice, I did not know about it! Thanks for making the change.